### ISSUE TITLE
custom bootscreen on 2.x

### ISSUE THREAD

**Issue by fred2088** [MarlinFirmware/Marlin#12219]
put the _bootscreen.h to \src\led
but need to put also with the _bootscreen.h on marline.ino folder

error
```
sketch\src\lcd\dogm/dogm_bitmaps.h:40:38: fatal error: ../../../_Bootscreen.h: No such file or directory
```
on Marlin 2.x bugfix download on 22/10/2018

**Comment by thinkyhead** [comment]
`_Bootscreen.h` must have the "B" capitalized and the file must be in the same folder as your Configuration files (and the `Marlin.ino` file).

**Comment by award0586** [comment]
Did this solve the problem, im having the same issue

**Comment by fred2088** [comment]
solved by using 2.0.5.3
